linux_os/guide/system/auditing/auditd_configure_rules/file_permissions_var_log_audit/ansible/shared.yml

```diff
@@ -1,62 +1,80 @@
-# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ol,multi_platform_rhv,multi_platform_sle
+# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ol,multi_platform_rhv,multi_platform_sle,multi_platform_ubuntu
 # reboot = false
 # strategy = restrict
 # complexity = low
 # disruption = low
 
-- name: Get audit log files
-  command: grep -iw ^log_file /etc/audit/auditd.conf
+- name: "{{{ rule_title }}} - Get Audit Log Files"
+  ansible.builtin.command: grep -iw ^log_file /etc/audit/auditd.conf
   failed_when: false
   register: log_file_exists
 
-- name: Parse log file line
-  command: awk -F '=' '/^log_file/ {print $2}' /etc/audit/auditd.conf
+- name: "{{{ rule_title }}} - Parse Log File Line"
```

Comment:
Given: https://github.com/ComplianceAsCode/content/blob/98aed892c28791c40556ec4cc3bd50ee8d73a094/linux_os/guide/system/auditing/auditd_configure_rules/file_group_ownership_var_log_audit/ansible/ubuntu.yml

Would be a good idea to reduce the command by using a `set_fact` by applying filters on `log_file_exists`.